src/main/java/me/karakelley/tictactoe/UserInterface.java

```diff
@@ -25,6 +26,16 @@ public String userPrompt(String message, Validator validator) {
       return input;
   }
 
+  public void waitForKeyPress() {
+    try {
+      io.anyKey();
+    } catch (IOException e) {
+      e.printStackTrace();
+    } catch (InterruptedException e) {
```

Comment:
You should never (unless there's a super great reason) swallow exceptions. What situations would cause these two exceptions to be thrown? In general there are two things you can do when an exception is thrown:

1. Handle it and do something about it
2. Throw it higher up the call stack